pkg/generate/ack/controller.go

```diff
@@ -57,6 +58,9 @@ var (
 		"Dereference": func(s *string) string {
 			return *s
 		},
+		"ToNames": func(s string) names.Names {
+			return names.New(s)
+		},
```

Comment:
Would it be possible to just use [`pkg/model.Field.Names`](https://github.com/aws-controllers-k8s/code-generator/blob/82c294c2e8fc6ba23baa0034520e84351bb7a32f/pkg/model/field.go#L35) instead?